content/providers/hardware.json

```diff
@@ -0,0 +1,162 @@
+[
+  {
+    "title": "IBM Qiskit Runtime",
+    "image": "static/images/providers/qiskit-ibm-runtime.png",
+    "description": "Qiskit Runtime is a quantum computing service and programming model that allows users to optimize workloads and efficiently execute them on quantum systems at scale. The programming model extends the existing interface in Qiskit with a set of new primitive programs.",
+    "installation": "pip install qiskit-ibm-runtime",
+    "websiteCta": {
+      "label": "Website",
+      "url": "https://quantum-computing.ibm.com/lab/docs/iql/runtime/"
+    },
+    "docCta": {
```

Comment:
I think `docsCta` is more semantic

Reply:
Fixed in https://github.com/Qiskit/qiskit.org/pull/2968/commits/60fb2c5174ff6e20fb77418e13bb75ca4fa0ea89